pyproject.toml

```diff
@@ -84,7 +84,6 @@ filterwarnings = [
     "ignore:Special cases found for .* but none were parsed.*:UserWarning",
     # end array_api_tests-related warnings
     "ignore:jax.extend.mlir.dialects.mhlo is deprecated.*:DeprecationWarning",
-    "ignore:jax.experimental.maps and .* are deprecated.*:DeprecationWarning",
```

Comment:
Looks like we need this warning suppression in the case of doctests. But I confirmed that all other tests passed without it.